spec.mediawiki

```diff
@@ -202,50 +204,87 @@ This section describes each of these design aspects.
 
 ===Fees===
 
-Rather than collecting fees on each trade, fees are collected by the pool during 
-registration only.
-Registration fees discourage certain spam attacks, and help to cover DEX 
-operating expenses.
-Further, collecting fees from trades executed via atomic swaps (where the server 
-is never in control of funds, and settlement occurs directly on-chain) would 
-add considerable complexity to the swap process.
-Registration fees will be configurable by the exchange operator, but the fee 
-would need to be reasonable for both the client and server operator.
-
-===Lot Sizes===
+====Trading Fees====
+
+The DEX collects no trading fees.
+Collecting fees from trades executed via atomic swaps (where the server 
+is never in control of funds and settlement occurs directly on-chain) would 
+add considerable complexity to the swap process and incentivize DEX operators to
+facilitate wash trading. 
+Instead, a one-time fee is collected by the pool during registration.
+Registration fees discourage certain spam attacks, enable punitive actions when 
+[[#Rules_of_Community_Conduct|conduct rules]] are violated, and help to cover 
+DEX operating expenses.
+Registration fees will be configurable by the exchange operator.
+
+====Transaction Fees====
+
+The clients will cover on-chain transaction fees at a fee rate set by the DEX 
+operator. 
```

Comment:
Not sure on this one. In some sense, the client is expected to pay exactly the specified rate since order validation assumes it and order completion may depend on it.

Reply:
Paying an exact rate reliably isn't really possible though because it's a fraction of atoms/bytes, integers where the bytes can't be controlled.

Even with dcrwallet set to use the standard 0.0001 DCR/kB, it rarely ends up that way.  Take for example these two transactions, each with two inputs and two outputs:
 - size: 417 bytes, fee: 4190 atoms, fee rate: 0.00010047 DCR/kB, https://explorer.dcrdata.org/tx/545ba9eb49b479d66e20cc025b1d106ec6a0ec168b23e69eaaa08db08ec21115
- size: 416 bytes, fee: 4190 atoms, fee rate: 0.00010072 DCR/kB, https://explorer.dcrdata.org/tx/ac345f94f6b43d039a83f6a0d699156723a2812f24261432579285a37757834d

Neither is exactly 0.0001 DCR/kB, and they aren't the same size anyway.